mbbsd/mail.c

```diff
@@ -1194,6 +1194,101 @@ mail_all(void)
     return 0;
 }
 
+void
+get_account_manager(struct Vector *namelist)
```

Comment:
There is two naming in this commit: account manager, and account sysop. Probably sticking to one of them would be better.